### ISSUE TITLE
Unhandled exception during Tab completion in Windows Terminal

### ISSUE THREAD

**Issue by brainplot** [PowerShell/PSReadLine#1684]
Hi, I'm using `PSReadLine` with the default PowerShell version that comes with Windows 10. As soon as I open a new PowerShell tab in the new Windows Terminal and I type `nvim<SPACE><TAB><TAB>`, I get the message below.

Interestingly, this only seems to happen if I'm in my `%USERPROFILE%` directory **and** I'm using the new Windows Terminal (using the normal PowerShell window seems fine).

`nvim` doesn't seem to be important for the bug to happen because it does the same thing with `subl` (i.e. the Sublime Text binary).
What I have noticed however is that when the error is about to happen, the first `<TAB>` expands to `.\` instead of just `.` (a dot). 
I recorded a video to show what I mean: https://streamable.com/c1d6dg

This is all the code I have in my `profile.ps1` as it pertains to `PSReadLine`. In case it matters, I've also included a list of all the plugins I have installed.
```powershell
if ($host.Name -eq 'ConsoleHost') {
	Import-Module PSReadLine

	Set-PSReadLineOption -EditMode Emacs
	Set-PSReadlineOption -BellStyle Visual
	Set-PSReadlineOption -HistorySearchCursorMovesToEnd

	Set-PSReadLineKeyHandler -Key Ctrl+w -Function BackwardKillWord
}
```

#### Error Message
```
-----------------------------------------------------------------------
Last 11 Keys:
 n v i m . e x e Space Tab Tab

Exception:
System.ArgumentOutOfRangeException: The value must be greater than or equal to zero and less than the console's buffer size in that dimension.
Parameter name: top
Actual value was -17.
   at System.Console.SetCursorPosition(Int32 left, Int32 top)
   at Microsoft.PowerShell.PSConsoleReadLine.Menu.EnsureMenuAndInputIsVisible(IConsole console, Int32 tooltipLineCount)
   at Microsoft.PowerShell.PSConsoleReadLine.Menu.DrawMenu(Menu previousMenu)
   at Microsoft.PowerShell.PSConsoleReadLine.PossibleCompletionsImpl(CommandCompletion completions, Boolean menuSelect)
   at Microsoft.PowerShell.PSConsoleReadLine.CompleteImpl(Boolean menuSelect)
   at Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics)
-----------------------------------------------------------------------
```

#### PowerShell Version
```
Major  Minor  Build  Revision
-----  -----  -----  --------
5      1      19041  1
```

#### Windows Terminal Version
`1.1.2021.0` (latest stable from the Microsoft Store, at the time of writing)

#### Installed Plugins
```

ModuleType Version    Name                                ExportedCommands                                             
---------- -------    ----                                ----------------                                             
Script     2.8.0      cd-extras                           {Clear-Stack, Expand-Path, Get-Ancestors, Get-Stack...}      
Manifest   3.1.0.0    Microsoft.PowerShell.Management     {Add-Computer, Add-Content, Checkpoint-Computer, Clear-Con...
Manifest   3.1.0.0    Microsoft.PowerShell.Utility        {Add-Member, Add-Type, Clear-Variable, Compare-Object...}    
Script     1.4.7      PackageManagement                   {Find-Package, Find-PackageProvider, Get-Package, Get-Pack...
Script     1.0.0      posh-git                            {Add-PoshGitToProfile, Expand-GitCommand, Format-GitBranch...
Script     0.0.1      posh-vcpkg                          TabExpansion                                                 
Script     2.2.4.1    PowerShellGet                       {Find-Command, Find-DscResource, Find-Module, Find-RoleCap...
Script     2.0.0      PSReadline                          {Get-PSReadLineKeyHandler, Get-PSReadLineOption, Remove-PS...
Script     1.4.2      ZLocation                           {Get-ZLocation, Invoke-ZLocation, Pop-ZLocation, Remove-ZL...
```

**Comment by vexx32** [comment]
Please update your version of PSReadLine (latest is 2.0.3) and give it another go. Many similar issues were fixed when PSRL 2.0 recently had the first stable release, and more since. 🙂

**Comment by brainplot** [comment]
Oh that fixed it. Thanks! Sorry for the noise caused by opening this issue! 😅
